### DISCUSSION TITLE
How to import file directly ?

### DISCUSSION THREAD

**Discussion by ManUtopiK** [nuxt/content#1990] [Q&A]
I want the raw content of an `.md` file.
I know I can import with Vite like this:
```js
import example from '../content/example.md?raw'
```
It works but it's not reactive in dev mode. This file is only loaded when nuxt start.

Any idea to do that in other way ?

**Comment by ManasMadrecha** [comment]
There's a `remark` plugin that lets you do this.
- https://github.com/remarkjs/remark/blob/main/doc/plugins.md
- https://github.com/unlight/remark-sources

**Reply by ManUtopiK** [reply]
Thanks for your response, but this is not what I'm looking for.
I just want to read the raw markdown, not transform it.

**Reply by ManasMadrecha** [reply]
That plugin doesn't transform anything; it simply outputs any file (incl. md) in place of the code.

You can simply write

```md
```../content/example.md
\```

```

This will be replaced by the raw contents of that file.
